tests/ui/attr-value-is-invalid.rs

```diff
@@ -0,0 +1,23 @@
+use bilge::prelude::*;
+
+// negative value
+#[bitsize(-1)]
+enum Test {}
+
+// one below lowest value
+// #[bitsize(0)]
+// enum Test {}
+
+// one above highest (struct) value
+#[bitsize(129)]
+struct Test {}
+
+// one above highest enum value
+#[bitsize(65)]
+enum Test {}
```

Comment:
now with #45, this fails with `empty enums are not supported`.

worth noting that the empty struct above it _does_ fail with the expected message. and the `#[bitsize("12")]` test below also fails as expected.
so we should probably change the order of checks in `crate::bitsize::analyze_enum`